bioconda_utils/cli.py

```diff
@@ -541,6 +541,9 @@ def dag(recipe_folder, config, packages="*", format='gml', hide_singletons=False
      help="""Bump package build numbers even if the only applicable pinning
      change is the python version. This is generally required unless you plan
      on building everything.""")
+@arg('--skip-variants',
```

Comment:
I added this as a convenience things so we can update pinnings for Python/R packages separately.

Reply:
However, those skips happen after the dependency graph is created, ergo packages that are dependencies of others may not be bumped.